src/test.rs

```diff
@@ -99,7 +99,9 @@ pub fn this_host_triple() -> String {
     } else {
         unimplemented!()
     };
-    let os = if cfg!(target_os = "linux") {
+    let os = if cfg!(target_os = "freebsd") {
+        "unknown-freebsd"
+    } else  if cfg!(target_os = "linux") {
```

Comment:
This change belongs in a different commit, also it's kinda covered by another PR which is trying to sort out FreeBSD CI etc.  Perhaps this would be best left out of this PR?

Reply:
Ok, I removed that part.